examples/fluids/problems/newtonian.c

```diff
@@ -16,47 +16,138 @@
 #include "../navierstokes.h"
 
 // For use with PetscOptionsEnum
-static const char *const StateVariables[] = {"CONSERVATIVE", "PRIMITIVE", "StateVariable", "STATEVAR_", NULL};
+static const char *const StateVariables[] = {"conservative", "primitive", "entropy", "StateVariable", "STATEVAR_", NULL};
```

Comment:
Why not stick with the allcaps? That's what we seem to do with everything else?

Reply:
Looks like HONEE has a mix of all-lower-case and mixed capitalization, but this is the only instance of allcaps.

I kind of prefer lower case since that's how most people will be writing out the command line options and it kinda helps with keeping readability in mind.

Reply:
I've added a commit to make everything consistently lower case.

Reply:
Yeah, that's why I made it consistently caps in Ratel so `STATEVAR_` + `CONSERVATIVE` etc is a valid enum value name.

Reply:
The documentation for `PetscOptionsEnum` says the array is just a list of options. Following through the code, the connection between the array and the enum is just an ordering thing; the `0` string in the array should refer to the enum value `0`.

I'll go ahead and make them all caps anyways though.